### DISCUSSION TITLE
Is it possible to use unattended w/o a product key?

### DISCUSSION THREAD

**Discussion by bendavis78** [memstechtips/UnattendedWinstall#118] [Q&A]
I volunteer for an education nonprofit that has received a large number of laptops from companies that are getting rid of them. We'd like to donate these laptops to students in the program, however these laptops have been completely wiped. I've been able to use ventoy to install the official Windows ISO, but there are still quite a few steps to set up a default user and install certain applications the students will need. I'd like to be able to create a USB installer that is fully automated to save time, but I have so far been unable to get unattend.xml to work w/o a product key. Is this even possible?

**Comment by memstechtips** [comment]
Hi there,

Yes, you don't need to enter a product key.
In my files, the product key is `<Key>00000-00000-00000-00000-00000</Key>` and that means it won't ask for a key.
